### ISSUE TITLE
36-7 regression. Driver level Antialiasing doesn't work anymore.

### ISSUE THREAD

**Issue by SpacePontif** [Beamdog/nwn-issues#544]
**To Reproduce**

 - Try to enable antialiasing through Nvidia Control Panel or Nvidia Profile Inspector.
 - Start the game, notice all the jaggies.

**Specifics**

 - OS: Windows 11
 - Game build: 8193.36-7
 - Distro platform: Steam
 - GPU (in case it may matter): RTX 4090
 - Content overrides? Yes, but I doubt they'd matter for this.
 - Regression? Yes. Antialiasing from the Nvidia Control Panel used to function in build 35.

The game has no built in AA so driver level AA is the only way to get it. This produces a pretty big downgrade in image quality compared to build 35.

**Comment by Soren217** [comment]
Strange, I have no issue getting it to work (also using Nvidia control panel), What are the exact settings you use?

And can others confirm?

**Comment by SpacePontif** [comment]
Here's the settings:
![image](https://github.com/Beamdog/nwn-issues/assets/153333985/f851e382-7f2e-4c32-8cda-60211f43e218)

After posting the thread I noticed I could get FXAA working, though it leaves a noticeable shimmer around moving edges, and I know I only used the other AA before 36-7.

**Comment by Soren217** [comment]
Could you try to disable MFAA?

**Comment by SpacePontif** [comment]
Just tried it, sadly didn't work. I also tested with every step of antialiasing (2x to 8x), every step of Antialiasing - Transparency (off, multisample, 2x to 8x), none of the settings made any kind of difference with the game running on version 36-7. 

I tested it on the current Stable build (35) and the settings all made a noticeable difference.

Using the settings posted above on build 35:
![image](https://github.com/Beamdog/nwn-issues/assets/153333985/3b2dad34-b29f-487a-b3c3-66943aaaf819)

On build 36-7 (very noticeable with movement, as the staircase effect creates a shimmer):
![image](https://github.com/Beamdog/nwn-issues/assets/153333985/d33f121f-32e9-409d-91ed-54df8f8f720d)
